### PR TITLE
SAM/BAM-to-BAM/SAM: remove duplicates, hide others

### PR DESCRIPTION
**Pull request by wm75**
In addition, both tools are still visible in the "SAM/BAM Manipulation" section, which is not populated from this file.

### CODE REVIEW

File: shed_tool_conf.xml [hunk 1]
```diff
@@ -64104,26 +64104,6 @@
             <version>1.4.0</version>
         </tool>
     </section>
-    <section id="sam_bam" name="SAM/BAM" version="">
```
Comment (bgruening) [inline shed_tool_conf.xml:64107]: warum löschen wir die? Reicht verstecken nicht einfach?

Comment (wm75) [reply shed_tool_conf.xml:64107]: There are exact duplicates of these records in the file. So I thought we could shrink it a tiny bit :)

Comment (bgruening) [reply shed_tool_conf.xml:64107]: mh, but f7a0d41036c7/sam_to_bam you deleted two times, no?

Comment (wm75) [reply shed_tool_conf.xml:64107]: yes, and for each record there is one more I'm keeping (started out with triplicated records)

Comment (wm75) [reply shed_tool_conf.xml:64107]: too bad the diff doesn't show the kept one

Comment (bgruening) [reply shed_tool_conf.xml:64107]: I see ok! Thanks.

**Pull request merged**